### PR TITLE
add control on empty bus bar list

### PR DESCRIPTION
**Pull request by bendaamerahm**

### CODE REVIEW

File: src/components/dialogs/input-hooks.js [hunk 1]
```diff
@@ -680,9 +681,18 @@ export const useExpandableValues = ({
     fieldProps,
     validateItem,
 }) => {
+    const useCostumStyle = makeStyles((theme) => ({
```
Comment (etiennehomer) [inline src/components/dialogs/input-hooks.js:684]: Costum or custom ?

File: src/components/dialogs/input-hooks.js [hunk 2]
```diff
@@ -680,9 +681,18 @@ export const useExpandableValues = ({
...
+            color: theme.palette.error.main,
+            textDecorationLine: 'overline',
+            margin: theme.spacing(1),
+        },
+    }));
+    const costumClasses = useCostumStyle();
     const classes = useStyles();
     const [values, setValues] = useState([]);
     const [errors, setErrors] = useState();
+    const [errorItem, setErrorItem] = useState(false);
```
Comment (etiennehomer) [inline src/components/dialogs/input-hooks.js:695]: something like : [isValuesListEmply, setValuesListEmpty] ?
This error is specific to the empty list problem

**Review by etiennehomer — CHANGES_REQUESTED** [review]

File: src/components/dialogs/input-hooks.js (hunk 1 above)
Comment (bendaamerahm) [reply src/components/dialogs/input-hooks.js:684]: my bad

File: src/components/dialogs/input-hooks.js (hunk 2 above)
Comment (bendaamerahm) [reply src/components/dialogs/input-hooks.js:695]: okey ill use [emptyListError, setEmptyListError]

File: src/components/dialogs/input-hooks.js (hunk 1 above)
Comment (etiennehomer) [reply src/components/dialogs/input-hooks.js:684]: That could be costum as it's come css. I add a doubt :)

File: src/components/dialogs/dialogUtils.js [hunk 3]
```diff
@@ -33,6 +33,11 @@ export const useStyles = makeStyles((theme) => ({
         fontSize: 'small',
         marginTop: theme.spacing(1),
     },
+    emptyListError: {
+        color: theme.palette.error.main,
+        textDecorationLine: 'overline',
```
Comment (etiennehomer) [inline src/components/dialogs/dialogUtils.js:38]: Remove overline maybe.
And the font size is too big. It should be like the other error messages

Comment (bendaamerahm) [reply src/components/dialogs/dialogUtils.js:38]: the other errors messages are default style of input control , ill remove the overline

File: src/components/dialogs/input-hooks.js [hunk 4]
```diff
@@ -282,29 +282,18 @@ export const useConnectivityValue = ({
...
-        let newConnectivityValue = {};
-        if (voltageLevelIdDefaultValue) {
-            const found_vl = voltageLevelOptions.find(
-                (value) => value.id === voltageLevelIdDefaultValue
-            );
-            if (found_vl !== undefined) {
-                newConnectivityValue.voltageLevel = found_vl;
-            } else {
-                newConnectivityValue.voltageLevel = {
-                    id: voltageLevelIdDefaultValue,
```
[the remark this answers is not in the document]
Comment (bendaamerahm) [reply src/components/dialogs/input-hooks.js:294]: no this is another hook

File: src/components/dialogs/input-hooks.js [hunk 5]
```diff
@@ -721,16 +713,26 @@ export const useExpandableValues = ({
...
 
     useEffect(() => {
         function validation() {
+            if (isRequired && values?.length === 0) {
+                setEmptyListError(true);
+                return;
+            }
             const res = validateItem(values);
+            if (isRequired && res?.size !== 0) {
+                setEmptyListError(true);
```
[the remark this answers is not in the document]
Comment (bendaamerahm) [reply src/components/dialogs/input-hooks.js:727]: good catch , im on it

**Review by sBouzols — APPROVED** [review]
Code Review OK
Tests OK

**Review by YenguiSeddik — APPROVED** [review]
code review: ok
Functional tests : ok

**Review by sBouzols — APPROVED** [review]
Ok Test OK

**Pull request merged**